static/scripts/calendar.js

```diff
@@ -49,6 +49,25 @@ const getCalendarLanguage = (langAttribute) => {
 	}
 };
 
+// Add data-testid attributes to navigation buttons in calendar page
+const addDatatestidsToCalendarNavigation = () => {
+	const buttonData = [
+		{ selector: '.fc-right .fc-prev-button', testId: 'right-prev-button' },
+		{ selector: '.fc-right .fc-next-button', testId: 'right-next-button' },
+		{ selector: '.fc-right .fc-today-button', testId: 'right-today-button' },
+		{ selector: '.fc-right .fc-timeGridDay-button', testId: 'right-day-view-button' },
+		{ selector: '.fc-right .fc-timeGridWeek-button', testId: 'right-week-view-button' },
+		{ selector: '.fc-right .fc-dayGridMonth-button', testId: 'right-month-view-button' },
+	];
+
+	buttonData.forEach(({ selector, testId }) => {
+		const button = document.querySelector(selector);
+		if (button) {
+			button.setAttribute('data-testid', testId);
```

Comment:
Better - but maybe not as readable... just a suggestion.
```suggestion
			button.dataset.testid = testId;
```

Reply:
done